cdap-ui/app/tracker/controllers/results-ctrl.js

```diff
@@ -137,6 +137,14 @@ class TrackerResultsController {
       scope: this.$scope
     };
 
+    if(params.query === '*') {
+      params.sort = 'creation-time desc';
+      params.numCursors = 10;
+    } else {
+      params.query = params.query + '*';
+      params.target = 'app';
```

Comment:
Sushil: It was just an example. The targets obviously represent what you are searching for. Please change as adviced by @elfenheart